.travis.yml

```diff
@@ -3,6 +3,9 @@ rvm:
   - 2.2.5
   - 2.3.1
 
+env:
+  - I3S_AUTH_FILE=examples/login_for_integration_i3s_tests.json
```

Comment:
Sure, I'll make it available in Travis once we get the PR to the merging point then we can drop it from here.